diff -U 3 dumb2wav.c dumb2wav.c
--- dumb2wav.c	Mon Apr 18 15:47:41 2022
+++ dumb2wav.c	Sun Apr 17 18:33:16 2022
@@ -22,6 +22,8 @@
 #include <stdio.h>
 #include <ctype.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <string.h>
 #include <dumb.h>
 
diff -U 3 dumbout.c dumbout.c
--- dumbout.c	Mon Apr 18 15:47:41 2022
+++ dumbout.c	Sun Apr 17 18:33:16 2022
@@ -22,6 +22,8 @@
 #include <stdio.h>
 #include <ctype.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <string.h>
 #include <dumb.h>
 
diff -U 3 clickrem.c clickrem.c
--- clickrem.c	Mon Apr 18 15:47:41 2022
+++ clickrem.c	Sun Apr 17 18:33:16 2022
@@ -19,6 +19,8 @@
 
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include "dumb.h"
 
 
diff -U 3 resample.c resample.c
--- resample.c	Mon Apr 18 15:47:41 2022
+++ resample.c	Sun Apr 17 18:33:16 2022
@@ -43,6 +43,8 @@
  */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include "dumb.h"
 
 
diff -U 3 itrender.c itrender.c
--- itrender.c	Mon Apr 18 15:47:41 2022
+++ itrender.c	Sun Apr 17 18:33:16 2022
@@ -18,6 +18,8 @@
  */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdlib.h>
 
 #include "dumb.h"
diff -U 3 read669.c read669.c
--- read669.c	Mon Apr 18 15:47:41 2022
+++ read669.c	Sun Apr 17 18:33:16 2022
@@ -20,6 +20,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "dumb.h"
 #include "internal/it.h"
diff -U 3 readasy.c readasy.c
--- readasy.c	Mon Apr 18 15:47:41 2022
+++ readasy.c	Sun Apr 17 18:33:16 2022
@@ -20,6 +20,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "dumb.h"
 #include "internal/it.h"
diff -U 3 readmod.c readmod.c
--- readmod.c	Mon Apr 18 15:47:41 2022
+++ readmod.c	Sun Apr 17 18:33:16 2022
@@ -20,6 +20,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "dumb.h"
 #include "internal/it.h"
diff -U 3 readmtm.c readmtm.c
--- readmtm.c	Mon Apr 18 15:47:41 2022
+++ readmtm.c	Sun Apr 17 18:33:16 2022
@@ -20,6 +20,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "dumb.h"
 #include "internal/it.h"
diff -U 3 readokt.c readokt.c
--- readokt.c	Mon Apr 18 15:47:41 2022
+++ readokt.c	Sun Apr 17 18:33:16 2022
@@ -20,6 +20,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "dumb.h"
 #include "internal/it.h"
diff -U 3 readoldpsm.c readoldpsm.c
--- readoldpsm.c	Mon Apr 18 15:47:41 2022
+++ readoldpsm.c	Sun Apr 17 18:33:16 2022
@@ -20,6 +20,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "dumb.h"
 #include "internal/it.h"
diff -U 3 readxm.c readxm.c
--- readxm.c	Mon Apr 18 15:47:41 2022
+++ readxm.c	Sun Apr 17 18:33:16 2022
@@ -20,6 +20,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "dumb.h"
 #include "internal/it.h"
diff -U 3 sequence.c sequence.c
--- sequence.c	Mon Apr 18 15:47:41 2022
+++ sequence.c	Sun Apr 17 18:33:16 2022
@@ -18,6 +18,8 @@
  */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdlib.h>
 #include <string.h>
 
