diff -U 3 gl_system.h gl_system.h
--- gl_system.h	Mon Apr 18 15:47:43 2022
+++ gl_system.h	Sat Apr 16 18:24:00 2022
@@ -72,7 +72,7 @@
 #include <OpenGL/OpenGL.h>
 #elif defined(__unix__)
 #include <GL/glew.h>
-#include "gl/api/glext.h"
+//#include "gl/api/glext.h"
 #else // !__APPLE__ && !__unix__
 #include <GL/gl.h>
 #include <GL/glu.h>
diff -U 3 gl_geometric.cpp gl_geometric.cpp
--- gl_geometric.cpp	Mon Apr 18 15:47:43 2022
+++ gl_geometric.cpp	Sun Apr 17 18:27:34 2022
@@ -1,6 +1,8 @@
 
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <float.h>
 #include "gl/utility/gl_geometric.h"
 
diff -U 3 gl_geometric.h gl_geometric.h
--- gl_geometric.h	Mon Apr 18 15:47:43 2022
+++ gl_geometric.h	Sun Apr 17 18:27:34 2022
@@ -2,6 +2,8 @@
 #define __GL_GEOM
 
 #include "math.h"
+#define REPLACE_WITH_AMDLIBM
+#include "amdlibm.h"
 #include "r_defs.h"
 
 class Vector
diff -U 3 opl.cpp opl.cpp
--- opl.cpp	Mon Apr 18 15:47:43 2022
+++ opl.cpp	Sun Apr 17 18:28:54 2022
@@ -28,6 +28,8 @@
 #include "../opl.h"
 #include "../muslib.h"
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include "m_random.h"
 
 static FRandom pr_opl;
diff -U 3 fmopl.cpp fmopl.cpp
--- fmopl.cpp	Mon Apr 18 15:47:43 2022
+++ fmopl.cpp	Sun Apr 17 18:28:54 2022
@@ -99,6 +99,8 @@
 #include <string.h>
 #include <stdarg.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 //#include "driver.h"		/* use M.A.M.E. */
 #include "opl.h"
 
diff -U 3 OPL3.cpp OPL3.cpp
--- OPL3.cpp	Mon Apr 18 15:47:43 2022
+++ OPL3.cpp	Sun Apr 17 18:28:54 2022
@@ -44,6 +44,8 @@
  */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <limits>
 
 #include "doomtype.h"
diff -U 3 opl_mus_player.cpp opl_mus_player.cpp
--- opl_mus_player.cpp	Mon Apr 18 15:47:43 2022
+++ opl_mus_player.cpp	Sun Apr 17 18:28:54 2022
@@ -4,6 +4,8 @@
 #include <string.h>
 #include <assert.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "opl_mus_player.h"
 #include "doomtype.h"
diff -U 3 colormaps.cpp colormaps.cpp
--- colormaps.cpp	Mon Apr 18 15:47:44 2022
+++ colormaps.cpp	Sun Apr 17 18:28:54 2022
@@ -35,6 +35,8 @@
 #include <stddef.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <float.h>
 
 #include "i_system.h"
diff -U 3 i_system.cpp i_system.cpp
--- i_system.cpp	Mon Apr 18 15:47:44 2022
+++ i_system.cpp	Sun Apr 17 18:30:16 2022
@@ -47,6 +47,8 @@
 
 #include "doomerrors.h"
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "SDL.h"
 #include "doomtype.h"
diff -U 3 i_sound.cpp i_sound.cpp
--- i_sound.cpp	Mon Apr 18 15:47:44 2022
+++ i_sound.cpp	Sun Apr 17 18:30:16 2022
@@ -59,6 +59,8 @@
 
 #include "doomtype.h"
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #ifndef NO_SOUND
 #include "fmodsound.h"
diff -U 3 music_dumb.cpp music_dumb.cpp
--- music_dumb.cpp	Mon Apr 18 15:47:44 2022
+++ music_dumb.cpp	Sun Apr 17 18:30:16 2022
@@ -22,6 +22,8 @@
 // HEADER FILES ------------------------------------------------------------
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include "i_musicinterns.h"
 #include "doomtype.h"
 #include "doomdef.h"
diff -U 3 thingdef_function.cpp thingdef_function.cpp
--- thingdef_function.cpp	Mon Apr 18 15:47:45 2022
+++ thingdef_function.cpp	Sun Apr 17 18:30:16 2022
@@ -38,6 +38,8 @@
 */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "tables.h"
 #include "tarray.h"
diff -U 3 instrum.cpp instrum.cpp
--- instrum.cpp	Mon Apr 18 15:47:45 2022
+++ instrum.cpp	Sun Apr 17 18:30:16 2022
@@ -27,6 +27,8 @@
 #include <string.h>
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "timidity.h"
 #include "m_swap.h"
diff -U 3 instrum_dls.cpp instrum_dls.cpp
--- instrum_dls.cpp	Mon Apr 18 15:47:45 2022
+++ instrum_dls.cpp	Sun Apr 17 18:30:16 2022
@@ -24,6 +24,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "timidity.h"
 #include "doomdef.h"
diff -U 3 instrum_sf2.cpp instrum_sf2.cpp
--- instrum_sf2.cpp	Mon Apr 18 15:47:45 2022
+++ instrum_sf2.cpp	Sun Apr 17 18:30:16 2022
@@ -2,6 +2,8 @@
 #include <string.h>
 #include <errno.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "doomdef.h"
 #include "m_swap.h"
diff -U 3 mix.cpp mix.cpp
--- mix.cpp	Mon Apr 18 15:47:45 2022
+++ mix.cpp	Sun Apr 17 18:30:16 2022
@@ -22,6 +22,8 @@
 */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdio.h>
 #include <stdlib.h>
 
diff -U 3 playmidi.cpp playmidi.cpp
--- playmidi.cpp	Mon Apr 18 15:47:45 2022
+++ playmidi.cpp	Sun Apr 17 18:30:16 2022
@@ -25,6 +25,8 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "timidity.h"
 #include "c_cvars.h"
diff -U 3 resample.cpp resample.cpp
--- resample.cpp	Mon Apr 18 15:47:45 2022
+++ resample.cpp	Sun Apr 17 18:30:16 2022
@@ -22,6 +22,8 @@
 */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdio.h>
 #include <stdlib.h>
 
diff -U 3 i_main.cpp i_main.cpp
--- i_main.cpp	Mon Apr 18 15:47:45 2022
+++ i_main.cpp	Sun Apr 17 18:31:12 2022
@@ -60,6 +60,8 @@
 #include <stdio.h>
 #include <stdarg.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #define USE_WINDOWS_DWORD
 #include "doomerrors.h"
diff -U 3 i_system.cpp i_system.cpp
--- i_system.cpp	Mon Apr 18 15:47:45 2022
+++ i_system.cpp	Sun Apr 17 18:31:12 2022
@@ -59,6 +59,8 @@
 #include "hardware.h"
 #include "doomerrors.h"
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "doomtype.h"
 #include "version.h"
diff -U 3 c_bind.cpp c_bind.cpp
--- c_bind.cpp	Mon Apr 18 15:47:42 2022
+++ c_bind.cpp	Sun Apr 17 18:25:02 2022
@@ -49,6 +49,8 @@
 #include "p_acs.h"
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdlib.h>
 
 /* Default keybindings for Doom (and all other games)
diff -U 3 c_cmds.cpp c_cmds.cpp
--- c_cmds.cpp	Mon Apr 18 15:47:42 2022
+++ c_cmds.cpp	Sun Apr 17 18:25:02 2022
@@ -35,6 +35,8 @@
 */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <time.h>
diff -U 3 c_console.cpp c_console.cpp
--- c_console.cpp	Mon Apr 18 15:47:42 2022
+++ c_console.cpp	Sun Apr 17 18:25:02 2022
@@ -37,6 +37,8 @@
 #include <stdarg.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdlib.h>
 #include <time.h>
 
diff -U 3 c_expr.cpp c_expr.cpp
--- c_expr.cpp	Mon Apr 18 15:47:42 2022
+++ c_expr.cpp	Sun Apr 17 18:25:02 2022
@@ -37,6 +37,8 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <string.h>
 
 #include "c_dispatch.h"
diff -U 3 d_main.cpp d_main.cpp
--- d_main.cpp	Mon Apr 18 15:47:42 2022
+++ d_main.cpp	Sun Apr 17 18:25:02 2022
@@ -47,6 +47,8 @@
 
 #include <time.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <assert.h>
 #include <sys/stat.h>
 
diff -U 3 d_netinfo.cpp d_netinfo.cpp
--- d_netinfo.cpp	Mon Apr 18 15:47:42 2022
+++ d_netinfo.cpp	Sun Apr 17 18:25:02 2022
@@ -33,6 +33,8 @@
 */
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdlib.h>
 #include <string.h>
 #include <assert.h>
diff -U 3 m_cheat.cpp m_cheat.cpp
--- m_cheat.cpp	Mon Apr 18 15:47:43 2022
+++ m_cheat.cpp	Sun Apr 17 18:25:02 2022
@@ -24,6 +24,8 @@
 
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "m_cheat.h"
 #include "d_player.h"
diff -U 3 network.cpp network.cpp
--- network.cpp	Mon Apr 18 15:47:43 2022
+++ network.cpp	Sun Apr 17 18:25:02 2022
@@ -66,6 +66,8 @@
 
 #include <ctype.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <list>
 #include "../GeoIP/GeoIP.h"
 
diff -U 3 networkheaders.h networkheaders.h
--- networkheaders.h	Mon Apr 18 15:47:43 2022
+++ networkheaders.h	Sun Apr 17 18:25:02 2022
@@ -98,6 +98,8 @@
 
 #include <ctype.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 // [BB] This is necessary under Linux, otherwise things like DWORD aren't defined.
 #ifndef __WIN32__
diff -U 3 networkshared.h networkshared.h
--- networkshared.h	Mon Apr 18 15:47:43 2022
+++ networkshared.h	Sun Apr 17 18:25:02 2022
@@ -60,6 +60,8 @@
 #include <time.h>
 #include <ctype.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 //--------------------------------------------------------------------------------------------------------------------------------------------------
 //-- DEFINES ---------------------------------------------------------------------------------------------------------------------------------------
diff -U 3 nodebuild.cpp nodebuild.cpp
--- nodebuild.cpp	Mon Apr 18 15:47:43 2022
+++ nodebuild.cpp	Sun Apr 17 18:25:02 2022
@@ -43,6 +43,8 @@
 #include <string.h>
 #include <stdio.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "doomdata.h"
 #include "nodebuild.h"
diff -U 3 p_glnodes.cpp p_glnodes.cpp
--- p_glnodes.cpp	Mon Apr 18 15:47:44 2022
+++ p_glnodes.cpp	Sun Apr 17 18:25:02 2022
@@ -31,6 +31,8 @@
 **
 */
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #ifdef _MSC_VER
 #include <malloc.h>		// for alloca()
 #endif
diff -U 3 p_map.cpp p_map.cpp
--- p_map.cpp	Mon Apr 18 15:47:44 2022
+++ p_map.cpp	Sun Apr 17 18:25:02 2022
@@ -24,6 +24,8 @@
 
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "templates.h"
 
diff -U 3 p_setup.cpp p_setup.cpp
--- p_setup.cpp	Mon Apr 18 15:47:44 2022
+++ p_setup.cpp	Sun Apr 17 18:25:02 2022
@@ -24,6 +24,8 @@
 
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #ifdef _MSC_VER
 #include <malloc.h>		// for alloca()
 #endif
diff -U 3 r_main.cpp r_main.cpp
--- r_main.cpp	Mon Apr 18 15:47:44 2022
+++ r_main.cpp	Sun Apr 17 18:25:02 2022
@@ -27,6 +27,8 @@
 
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 // [BB] network.h has to be included before stats.h under Linux.
 // The reason should be investigated.
diff -U 3 r_polymost.cpp r_polymost.cpp
--- r_polymost.cpp	Mon Apr 18 15:47:44 2022
+++ r_polymost.cpp	Sun Apr 17 18:25:02 2022
@@ -69,6 +69,8 @@
 #include <stdlib.h>
 #include <assert.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include "doomtype.h"
 #include "r_polymost.h"
 #include "c_cvars.h"
diff -U 3 r_utility.cpp r_utility.cpp
--- r_utility.cpp	Mon Apr 18 15:47:44 2022
+++ r_utility.cpp	Sun Apr 17 18:25:02 2022
@@ -27,6 +27,8 @@
 
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "templates.h"
 #include "doomdef.h"
diff -U 3 tables.h tables.h
--- tables.h	Mon Apr 18 15:47:44 2022
+++ tables.h	Sun Apr 17 18:25:03 2022
@@ -38,6 +38,8 @@
 
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include "basictypes.h"
 
 #ifndef PI
diff -U 3 v_font.cpp v_font.cpp
--- v_font.cpp	Mon Apr 18 15:47:45 2022
+++ v_font.cpp	Sun Apr 17 18:25:03 2022
@@ -77,6 +77,8 @@
 #include <stdio.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <ctype.h>
 
 #include "templates.h"
diff -U 3 v_palette.cpp v_palette.cpp
--- v_palette.cpp	Mon Apr 18 15:47:45 2022
+++ v_palette.cpp	Sun Apr 17 18:25:03 2022
@@ -35,6 +35,8 @@
 #include <stddef.h>
 #include <string.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <float.h>
 #ifdef _WIN32
 #include <io.h>
diff -U 3 vectors.h vectors.h
--- vectors.h	Mon Apr 18 15:47:45 2022
+++ vectors.h	Sun Apr 17 18:25:03 2022
@@ -41,6 +41,8 @@
 #define VECTORS_H
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <string.h>
 
 #ifndef PI
diff -U 3 zstrformat.cpp zstrformat.cpp
--- zstrformat.cpp	Mon Apr 18 15:47:45 2022
+++ zstrformat.cpp	Sun Apr 17 18:25:03 2022
@@ -86,6 +86,8 @@
 #include <stddef.h>
 #include <stdlib.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <stdlib.h>
 #include <locale.h>
 
