diff -U 3 collector.cpp collector.cpp
--- collector.cpp	Mon Apr 18 15:47:45 2022
+++ collector.cpp	Sun Apr 17 18:39:14 2022
@@ -58,6 +58,8 @@
 #include "network.h"
 
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 #include <time.h>
 
 //--------------------------------------------------------------------------------------------------------------------------------------------------
diff -U 3 network.cpp network.cpp
--- network.cpp	Mon Apr 18 15:47:45 2022
+++ network.cpp	Sun Apr 17 18:39:14 2022
@@ -58,6 +58,8 @@
 
 #include <ctype.h>
 #include <math.h>
+#define REPLACE_WITH_AMDLIBM
+#include <amdlibm.h>
 
 #include "../src/huffman/huffman.h"
 #include "main.h"
